src/Font.ts

```diff
@@ -0,0 +1,159 @@
+ *
+ * The `font-family` CSS property specifies a prioritized list of one or more
+ * font family names and/or generic family names for the selected element.
+ *
+ * @since 1.0.0
+ */
+export type FontFamily = string
+
+/**
+ * Represents optional values for modifying the style of a font.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/CSS/font
+ *
+ * @since 1.0.0
+ */
+export interface FontOptions {
+  /**
+   * Represents the `font-style` CSS property.
+   *
+   * The `font-style` CSS property sets whether a font should be styled with a normal,
+   * italic, or oblique face from its * font-family.
+   */
+  readonly style: O.Option<string>
+
+  /**
+   * Represents the `font-variant` CSS property.
+   *
+   * The `font-variant` CSS property is a shorthand for the longhand properties `font-variant-caps`,
+   * `font-variant-numeric`, `font-variant-alternates`, `font-variant-ligatures`, and `font-variant-east-asian`.
+   */
+  readonly variant: O.Option<string>
+
+  /**
+   * Represnts the `font-weight` CSS property.
+   *
+   * The `font-weight` CSS property sets the weight (or boldness) of the font.
+   */
+  readonly weight: O.Option<string>
+}
+
+/**
+ * Constructs a new `FontOptions` object.
+ *
+ * @since 1.0.0
+ */
+export const fontOptions = ({
+  style,
+  variant,
+  weight
+}: {
+  readonly style?: string
+  readonly variant?: string
+  readonly weight?: string
+}): FontOptions => ({
+  style: O.fromNullable(style),
+  variant: O.fromNullable(variant),
+  weight: O.fromNullable(weight)
+})
+
+/**
+ * The `Show` instance for `FontOptions`.
+ *
+ * @since 1.0.0
+ */
+export const ShowFontOptions: S.Show<FontOptions> = {
+  show: (o) => intercalate(monoidString, ROR.readonlyRecord)(' ', ROR.compact(ROR.fromRecord(o)))
+}
+
+/**
+ * Shows the string representation of `FontOptions`.
+ *
+ * @since 1.0.0
+ */
+export const showFontOptions = ShowFontOptions.show
+
+/**
+ * Represents the `font` CSS property.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/CSS/font
+ *
+ * @since 1.0.0
+ */
+export interface Font {
+  /**
+   * Represents the `font-family` CSS property.
+   *
+   * The `font-family` CSS property specifies a prioritized list of one or more
+   * font family names and/or generic family names for the selected element.
+   */
+  readonly fontFamily: FontFamily
+
+  /**
+   * Represents the `font-size` CSS property.
+   *
+   * The `font-size` CSS property sets the size of the font.
+   */
+  readonly size: number
+
+  /**
+   * Represents optional values for modifying the style of a font.
+   */
+  readonly fontOptions: FontOptions
+}
+
+/**
+ * Constructs a new `Font`.
+ *
+ * @since 1.0.0
+ */
+export const font = (fontFamily: FontFamily, size: number, options?: FontOptions): Font => ({
+  fontFamily,
+  size,
+  fontOptions: typeof options === 'object' ? options : fontOptions({})
+})
+
+/**
+ * The `Show` instance for `Font`.
+ *
+ * @since 1.0.0
+ */
+const ShowFont: S.Show<Font> = {
```

Comment:
idem, should be `showFont`

Reply:
See https://github.com/gcanti/graphics-ts/pull/7/commits/7898462206775283d6538be112aef83ceac2e28f